.env.example

```diff
@@ -1,15 +1,13 @@
-# for nginx reverse proxy to work through the containers, the bancho
-# server must expose itself on port 80 to be accessed on http://bancho.
-SERVER_ADDR=0.0.0.0
-SERVER_PORT=80
+APP_HOST=0.0.0.0
+APP_PORT=80
```

Comment:
Shouldn't this port be changed so it does not potentially conflict with nginx if exposed?

Reply:
this port is internal to the container - just pushed up a fix to expose it on 10000 & use that in nginx on host machine

Reply:
perhaps this APP_PORT should represent the externally exposed port, since 80 should actually be const for port inside the container

Reply:
perhaps they should be the same inside & outside of the container for even more simplicity

Reply:
ok addressed in latest commit

Reply:
> perhaps they should be the same inside & outside of the container for even more simplicity

yeah this is what i was trying to say, mb